### ISSUE TITLE
Fish allergy and shell fish allergy

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11123]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
